LayoutTests/imported/w3c/web-platform-tests/import-maps/nonimport-integrity-expected.txt

```diff
@@ -0,0 +1,16 @@
+CONSOLE MESSAGE: Cannot load script http://localhost:8800/import-maps/resources/log.js?pipe=sub&name=ModuleNoIntegrity. Failed integrity metadata check. Content length: 35, Expected content length: 35, Expected metadata: sha384-foobar
+CONSOLE MESSAGE: TypeError: Cannot load script due to integrity mismatch
+CONSOLE MESSAGE: Cannot load script http://localhost:8800/import-maps/resources/log.js?pipe=sub&name=ModuleBadIntegrityAttribute. Failed integrity metadata check. Content length: 45, Expected content length: 45, Expected metadata: sha384-foobar
+CONSOLE MESSAGE: TypeError: Cannot load script due to integrity mismatch
+
+PASS Script was not loaded as its integrity check was not ignored
+PASS Script was loaded as its correct integrity attribute was not ignored
+PASS Script was loaded as its empty integrity attribute was not ignored
+PASS Script was not loaded as its bad integrity attribute was not overridden
+FAIL Modulepreload was not loaded as its integrity check was not ignored assert_unreached: Should have rejected: undefined Reached unreachable code
+PASS Modulepreload was loaded as its correct integrity attribute was not ignored
+PASS Modulepreload was loaded as its empty integrity attribute was not ignored
+FAIL Modulepreload was not loaded as its bad integrity attribute was not ignored assert_unreached: Should have rejected: undefined Reached unreachable code
```

Comment:
This fails because module preload integrity is [not supported](https://bugs.webkit.org/show_bug.cgi?id=238206).